pkgs/top-level/aliases.nix

```diff
@@ -380,7 +380,6 @@ mapAliases ({
   virtviewer = virt-viewer; # added 2015-12-24
   vorbisTools = vorbis-tools; # added 2016-01-26
   webkit = webkitgtk; # added 2019-03-05
-  weechat-xmpp = weechatScripts.weechat-xmpp; # added 2018-09-06
```

Comment:
Maybe?

```
weechat-xmpp = throw "weechat-xmpp has been removed as it doesn't receive
     any updates from upstream and depends on outdated Python2-based modules.";
```

Reply:
Not in favour. Removal already throws an error, and the reason for removal is documented in the release notes.